src/Common.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2020 PTC Inc.
```

Comment:
Suggestion: Forgot to mention that I added info to the README about using [SPDX identifiers](https://spdx.org/ids) to clean up the license texts if you want to use it on your new files (see include/CMakeList.txt files for example).

This whole comment would become:

```cpp
// SPDX-License-Identifier: BSL-1.0
// Copyright 2020 PTC Inc.
```